docker-compose.yml

```
@@ -44,7 +44,7 @@ services:
   homeassistant:
     container_name: homeassistant
     restart: unless-stopped
-    image: "homeassistant/home-assistant:2023.2"
+    image: "homeassistant/home-assistant:2023.3"
```

Let's make it more strict
```suggestion
    image: "homeassistant/home-assistant:2023.3.5"
```